components/TestimonialList/index.js

```diff
@@ -22,6 +23,22 @@ const StyledSupporter = Supporter.extend`
   }
 `
 
+const TestimonialText = styled(Markdown)`
+  p:first-child::before {
+    display: inline;
+    /* Use arrow function to ensure translation happens dynamically
+     * (at not just once at component defintion time) */
```

Comment:
-> "and not..."
-> "definition time"

Reply:
Fixed!

Reply:
Forget to push?

Reply:
Forgot to stage even 😅 Fixed for good now.